From gernot@hillier.de  Fri Jan 15 14:45:35 2010
From: Gernot Hillier <gernot@hillier.de>
Date: Mon, 11 Jan 2010 09:06:44 +0100
Subject: USB: serial: option.c: Add chipset information for 4G W14
To: Matthias Urlichs <smurf@smurf.noris.de>, Greg Kroah-Hartman <gregkh@suse.de>, Oliver Neukum <oliver@neukum.org>
Message-ID: <4B4ADC14.3060006@hillier.de>


From: Gernot Hillier <gernot@hillier.de>

Carsten Juttner thankfully investigated a bit and found out some details
about the chipset used in the 4G W14 device I recently added to
option.c.

I think this information is useful for reference, so I'd be happy if you
could include those bits.

Signed-off-by: Gernot Hillier <gernot@hillier.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/option.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -338,6 +338,8 @@ static int  option_resume(struct usb_ser
 
 /* 4G Systems products */
 #define FOUR_G_SYSTEMS_VENDOR_ID		0x1c9e
+/* This is the 4G XS Stick W14 a.k.a. Mobilcom Debitel Surf-Stick *
+ * It seems to contain a Qualcomm QSC6240/6290 chipset            */
 #define FOUR_G_SYSTEMS_PRODUCT_W14		0x9603
 
 /* Haier products */
